lib/app/modules/home/views/home_view.dart

```diff
@@ -308,12 +310,17 @@ class HomeView extends GetView<HomeController> {
                     axisDirection: AxisDirection.down,
                     child: Obx(() {
                       return FutureBuilder(
-                          future:
-                              controller.initStream(controller.userModel.value),
+                          future: controller.isSortedAlarmListEnabled.value
+                              ? controller
+                                  .initStream(controller.userModel.value)
+                              : controller
+                                  .initStream(controller.userModel.value),
                           builder: (context, AsyncSnapshot snapshot) {
                             if (snapshot.hasData) {
                               final Stream streamAlarms = snapshot.data;
 
+                              print("DKNDNVKDVNDKDVND");
```

Comment:
I understand we have to debug, but could you remove/change to something more meaningful?

Reply:
Yaa, sure! I'll be removing that comment. Sorry for the inconvenience.